src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int LIMIT_OF_NUMBERS = 101;
```

Comment:
I think a wise approach is to separate business aspects (human logic) of code from technical ones. In this case - a business requirement is to assign a number up to 100 - so in my opinion, a constant should be assigned with value 100 instead of 101.
Then, a technical requirement is to provide a maximum value which will be used by `Random` object to select a number. In our case it will be `101` so when calling `nextInt()`, so you provide a constant + 1.
It is also important to provide a good, self-explanatory name for every constant and variable, in this case I think it would be better to name it "MAX_NUMBER".

Reply:
Thank you for your tips and explanation.